### PR TITLE
Sprint-6-Backend

### PR DESCRIPTION
**Pull request by Waldleufer**

### CODE REVIEW

**Review by SaiVarunVaranasi — APPROVED** [review]
looks good. Need to work on the build deployement.

File: backend/Test/UnitTest1.cs [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+             Assert.True(apiResponse.IsSuccessStatusCode);
+
+             var stringResponse = await apiResponse.Content.ReadAsStringAsync();
+
+             dynamic result = JsonConvert.DeserializeAnonymousType(stringResponse, new { message = "" });
+             Assert.Equal("Hello API", result.message);
+         }
+        
+    }*/
+    }
```
Comment (MartinDuersch) [inline backend/Test/UnitTest1.cs:89]: Can be used to add more customised unit tests in the future

**Review by MartinDuersch — APPROVED** [review]
Looks good :)

**Comment by Waldleufer** [comment]
Was not merged because if wasn't complete.

**Pull request closed without merging**